cmd/inspect.go

```diff
@@ -92,6 +92,25 @@ func (cli *CLI) inspect(opts Options, args []string) int {
 	cli.formatter.Print(issues, nil, cli.sources)
 
 	if len(issues) > 0 && !force {
+		if opts.MinimumSeverity != "" {
+			var minSeverity tflint.Severity
+			switch strings.ToLower(opts.MinimumSeverity) {
+			case "warning":
+				minSeverity = 1
+			case "error":
+				minSeverity = 2
```

Comment:
This should be handled alongside the severity type, e.g. `func NewSeverity(s string) (Severity, error)`. An error would be returned if `s` is not a recognized severity. Handling it here is too error prone, e.g. if a new severity is added someone has to find and update this code.

Reply:
I guess with the addition of `choice` to the struct tag an update is required anyway, but it would still be good to not hardcode this. At a minimum these severity codes should be exported/referenced instead of directly hardcoding the ints.

Reply:
Like I mentioned above I think adding a new severity would be a task requiring many changes anyway, but I do agree the logic around severity should not be in inspect.go. I the functions which handle the conversion from the string into Severity and into in32 for comparison into the issue.go file. Not sure if this is the best space for it, but that seemed to be the home of severity.
I also removed the redeclaration of severity and changed references to point to the declaration in the SDK.